content/en/docs/configuration/profiles.md

```diff
@@ -95,9 +96,9 @@ The order of application of profiles is therefore:
 Profiles are therefore a composable way to specify configuration subsets in a contextual manner.
 
 {{% blocks/callout type="info" title="Locking Dependencies" %}}
- Only dependencies list at the top level of `rebar.config`, the `default` profile,  are saved to `rebar.lock`. Other dependencies will not get locked.
+Only dependencies listed at the top level of `rebar.config`, the `default` profile, are saved to `rebar.lock`. Other dependencies will not get locked.
 
-If someone wants to "lock for production" (meaning with production-related profiles), the answer is to keep the default profile and to use [releases](/docs/deployment/releases), which allow to produce compiled artifacts that can be reused at any time.
+If someone wants to "lock for production" (meaning with production-related profiles), the answer is to keep the default profile and to use [releases](/docs/deployment/releases), which allows producing compiled artifacts that can be reused at any time.
```

Comment:
```suggestion
If someone wants to "lock for production" (meaning with production-related profiles), the answer is to keep the default profile and to use [releases](/docs/deployment/releases), which produce compiled artifacts that can be reused at any time.
```

I think this whole `x allows y-ing` is my broken English and there's generally a better way to go about it. No need to review it all, but I'll catch and suggest those I see as I go.

Reply:
aae0f90